src/main/java/longah/handler/StorageHandler.java

```diff
@@ -141,16 +141,23 @@ public void loadTransactionsData() throws LongAhException {
 
                 String[] transactionData = data.split(SEPARATOR);
                 String lenderName = transactionData[0];
+                String transactionTime = null;
                 Member lender = members.getMember(lenderName);
                 ArrayList<Subtransaction> subtransactions = new ArrayList<>();
 
+                if (transactionData[transactionData.length -1].contains("-")) {
```

Comment:
Might be better to detect by using even/odd parity of the length. If no time object exists should have odd numbered index, vice versa (verify this before implementing). Also referencing length - 1 is not very elegant, might be better to have time come before borrowers (in toStorage)

Reply:
Resolved! transaction time is now the second component in the expression